tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/species_features.tsx

```diff
@@ -12,7 +12,14 @@ export const facial_hair_color: Feature<string> = {
 
 export const facial_hair_gradient: FeatureChoiced = {
   name: 'Facial hair gradient',
-  component: FeatureDropdownInput,
+  component: (props) => {
+    const conf = {
+      ...props,
+      buttons: true,
+    };
+
+    return FeatureDropdownInput(conf);
```

Comment:
```suggestion
    return <FeatureDropdownInput buttons {...props} />
```

Or whatever the syntax is, but you must never call a functional component.

Also why are you repeating this code over and over again? I thought this is why you had the separate Feature.

Reply:
I moved feature in default dropdown how optional and turn on its where needed

Reply:
Okay, that's fine, but then you should be using it like this